setup.py

```diff
@@ -86,6 +88,11 @@ def has_ext_modules(self):
     packages=find_packages(),
     ext_modules=get_ext_modules(),
     install_requires=Path("requirements.txt").read_text().splitlines(),
+    extras_require={
+        "tensorflow": [f"tensorflow>={min_tf_version},<{max_tf_version}"],
+        "tensorflow-gpu": [f"tensorflow-gpu>={min_tf_version},<{max_tf_version}"],
+        "tensorflow-cpu": [f"tensorflow-cpu>={min_tf_version},<{max_tf_version}"],
```

Comment:
This is failing CI and flake. Since we support py35+ could you use `.format` instead of f-strings

Reply:
Also please merge with the latest master branch when you get a chance and it'll fix the other fails

Reply:
Updated.  Thanks!